src/components/footer/footer.js

```diff
@@ -1,60 +1,81 @@
+  LINK_FACEBOOK,
+  LINK_INSTAGRAM,
+  LINK_YOUTUBE,
+} = FOOTER;
 
-const links = [
-  { id: 0, name: "Home", path: "/home" },
-  { id: 1, name: "Search Event", path: "/events" },
-  { id: 2, name: "Create Event", path: "" },
-  { id: 3, name: "Terms", path: "/terms" },
-  { id: 4, name: "About", path: "/about" },
+const LiNKS = [
+  { path: PATH_HOME, name: HOME },
+  { path: PATH_EVENTS, name: SEARCH_EVENT },
+  { path: PATH_CREATE_EVENT, name: CREATE_EVENT },
+  { path: PATH_TERMS, name: TERMS },
+  { path: PATH_ABOUT, name: ABOUT },
 ];
 
-const socialLinks = [
-  { id: 0, icon: "fab fa-facebook-f", link: "https://uk-ua.facebook.com" },
-  { id: 1, icon: "fab fa-instagram", link: "https://www.instagram.com" },
-  { id: 2, icon: "fab fa-youtube", link: "https://www.instagram.com" },
+const SOCIALLINKS = [
+  { icon: STYLE_ICON_FACEBOOK, link: LINK_FACEBOOK },
+  { icon: STYLE_ICON_INSTAGRAM, link: LINK_INSTAGRAM },
+  { icon: STYLE_ICON_YOUTUBE, link: LINK_YOUTUBE },
 ];
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_container">
+        <NavLink to={HOME}>
+          <img src={LogoWhite} alt="Logo" className="footer__logo" />
+        </NavLink>
+        <div className="footer__links">
+          {LiNKS.map(({ path, name }) => (
+            <NavLink to={path} key={name} className="footer__link_item">
+              {name}
+            </NavLink>
+          ))}
+        </div>
       </div>
-      <div className="custom-footer__subscribe">
+      <div className="footer__subscribe">
         <h4>{SUBSCRIBE_US}</h4>
-        <form>
+        <form className="footer__form">
           <input
             type="email"
-            value="Paste your email"
-            className="custom-footer__subscribe i"
-          ></input>
-          <button type="submit" className="btn-dark-theme">
-            {SUBSCRIBE}
-          </button>
+            placeholder={PASTE_YOUR_EMAIL}
+            className="footer__input"
+          />
+          <IoMdSend className="footer__subscribe__icon" />
         </form>
       </div>
-      <div className="social-links">
+      <div className="footer__social_links">
         <h4>{CONTACT_US}</h4>
         <p>{PHONE_NUMBER}</p>
-        <div className="social-links__icons">
-          {socialLinks.map(link => (
-            <Link key={link.id} to={link.link} className="nav-link social-link">
-              <i className={link.icon} />
-            </Link>
+        <div className="footer__social_links_icons">
+          {SOCIALLINKS.map(({ link, icon }) => (
```

Comment:
As well as for LINKS.map, please move it outside of component

Reply:
Done